index.html

```diff
@@ -275,12 +275,12 @@
             for a subsequent requestPointerLock.
           </p>
 
-          <p>If any element (including this one) in the same document
+          <p>If any element (including this one) in the same shadow-including document
           is already locked (or pending lock) the pointer
           lock <a>target</a> must be updated to this element and a
           <a>pointerlockchange</a> event sent.</p>
 
-          <p>If any element in another document is already locked the request
+          <p>If any element in another shadow-including document is already locked the request
```

Comment:
This has a similar problem to the above one @hayatoito noted. Although it is probably not a big deal since I think any reader can figure out what it means. But IMO it would be clearer to talk about roots explicitly:

"If any element whose shadow-including root is a different document is already locked, the request must fail..."